packaging/debian/create_package.sh

```diff
@@ -47,6 +47,8 @@ do
     install -m 0644 $basedir/packaging/Linux/$f.service $systemddir/$f.service;
     install -m 0644 $basedir/packaging/Linux/$f.socket $systemddir/$f.socket;
 done
+install -m 0644 $basedir/packaging/Linux/albatross_influx.service \
```

Comment:
Thinking aloud: We don't need `albatross_influx.socket` because albatross-influx does not listen for connections. Instead, it connects to the stats daemon and forwards the stats to influx.